test/rules/assertions/checkParamNames.js

```diff
@@ -250,7 +267,7 @@ export default {
            * @param [cfg.foo]
            * @param baz
            */
-          function quux ({foo, bar}, baz) {
+          function quux ({foo}, baz) {
```

Comment:
Also here...

Reply:
This change removed the `bar` property since the test was for duplicate properties, not non-existent properties. Leaving it as `{foo, bar}` results in the test failing and only reporting the missing property.